content/spin/v2/rust-components.md

```diff
@@ -12,6 +12,7 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/rust-compo
 - [Redis Components](#redis-components)
 - [Sending Outbound HTTP Requests](#sending-outbound-http-requests)
 - [Routing in a Component](#routing-in-a-component)
+- [Granular Routing](#granular-routing)
```

Comment:
Oh sorry this isn't a Rust thing it's at the Spin manifest level.  It should be an extra paragraph in https://developer.fermyon.com/spin/v2/http-trigger#http-trigger-routes (with a mention in https://developer.fermyon.com/spin/v2/http-trigger#additional-request-information for how to access the match info).  I can send a PR for it if you like?